### PR TITLE
Add homebrew for tea

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

File: tea.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+require "formula"
+
+class Gitea < Formula
+  homepage "https://github.com/go-gitea/tea"
+  head "https://github.com/go-gitea/tea.git"
+
+  stable do
+    version "1.0.0"
```
Comment (techknowlogick) [inline tea.rb]: TODO: tag a version for tea

File: tea.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
+require "formula"
+
+class Gitea < Formula
```
Comment (lunny) [inline tea.rb]: Tea?

**Review by lunny — CHANGES_REQUESTED** [review]

File: tea.rb (hunk 2 above)
Comment (techknowlogick) [reply tea.rb]: Changed. Thanks for noticing :)

File: tea.rb (hunk 1 above)
Comment (lunny) [reply tea.rb]: Since `tea` is too early I would like name it version `0.1.0`.

**Review by lunny — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Pull request merged**